NSS-Day-Cohort-42/rare-API-media-moguls
# Users can now edit tags.

#### Changes Made
1. Modified file `tags.py` in `rareapi/views` dir to include edit functionality.
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-edit-tag
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality in Postman.
> Create GET request with url `http://localhost:8000/tags`
> In a new tab, create a PUT request with URL `http://localhost:8000/tags`, and paste the following code into the 'Body' > 'Raw' section:
```
{
    "id": 1,    
    "label": "testing123"
}
```
> Make sure the dropdown is set to `JSON`, and send the request.
> Run your GET request and view your newly updated tag.
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
